README.md

```diff
@@ -31,17 +31,27 @@ kubectl -n fybrik-system label configmap sample-policy openpolicyagent.org/polic
 while [[ $(kubectl get cm sample-policy -n fybrik-system -o 'jsonpath={.metadata.annotations.openpolicyagent\.org/policy-status}') != '{"status":"ok"}' ]]; do echo "waiting for policy to be applied" && sleep 5; done
 ```
 
-### Deploy Fybrik application which triggers the module
+### Deploy Fybrik application
 ```bash
 kubectl apply -f fybrikapplication.yaml -n default
 ```
-Run the following command to wait until the fybrikapplication be ready.
+Wait for the fybrik module:
 ```bash
-while [[ $(kubectl get fybrikapplication my-notebook -n default -o 'jsonpath={.status.ready}') != "true" ]]; do echo "waiting for FybrikApplication" && sleep 5; done
+while [[ ($(kubectl get fybrikapplication my-notebook -n default -o 'jsonpath={.status.ready}') != "true") || ($(kubectl get jobs my-notebook-default-dremio-module -n fybrik-blueprints -o 'jsonpath={.status.conditions[0].type}') != "Complete") ]]; do echo "waiting for FybrikApplication" && sleep 5; done
```

Comment:
we can add a comment here, that the operation can take several minutes

Reply:
Dremio doesn't workk for me

fybrik-blueprints    dremio-master-0                              0/1     Init:3/4   0          3m48s